Content.IntegrationTests/Tests/Damageable/DamageableTest.cs

```diff
@@ -53,14 +53,13 @@ public class DamageableTest : ContentIntegrationTest
     - TestDamage3b
     - TestDamage3c
 
-- type: resistanceSet
+- type: damageModifierSet
   id: testResistances
 # this space is intentionally left blank
 
```

Comment:
```suggestion
```
I accidentally left this in. Unless you now make use of the `testResistances` in a test somewhere, this can probably be deleted